src/server/config.py

```diff
@@ -1,21 +1,30 @@
 import os
 import sqlalchemy as db
+from sqlalchemy_utils import database_exists, create_database
 
 # Determine if app is ran from docker or local by env var IS_LOCAL
 BASE_PATH = '/app/static/'
-DB = 'postgresql://postgres:thispasswordisverysecure@paws-compose-db/postgres'
+POSTGRES_PASSWORD=os.getenv('POSTGRES_PASSWORD', 'thispasswordisverysecure')
+POSTGRES_DATABASE=os.getenv('POSTGRES_DATABASE', 'paws')
+POSTGRES_USER=os.getenv('POSTGRES_USER', 'postgres')
+DB = 'postgresql://' + POSTGRES_USER + ':'+ POSTGRES_PASSWORD + '@paws-compose-db/' + POSTGRES_DATABASE
 
 IS_LOCAL = os.getenv('IS_LOCAL', False)
 
 if IS_LOCAL:
     BASE_PATH = '../local_files/'
     DB = os.getenv('LOCAL_DB_IP',
-                   'postgresql://postgres:thispasswordisverysecure@localhost:5432/postgres')
+                   'postgresql://postgres:'+ POSTGRES_PASSWORD + '@localhost:5432/' + POSTGRES_DATABASE)
 
 #best practices is to have only one engine per application process
 #https://docs.sqlalchemy.org/en/13/core/connections.html
 engine = db.create_engine(DB)
 
+if not database_exists(engine.url):
```

Comment:
why do we need the create_database method here?
@dkelley

Reply:
Because only the "postgres" db is auto created (along with the templates), so if we want to use our own db (a good practice) we need to create the database in order for the create statements to work later in the process.  For example we can now use the paws_db or whatever is an appropriate name.